lldb/source/Plugins/Platform/WebAssembly/PlatformWasmProperties.td

```diff
@@ -0,0 +1,19 @@
+include "../../../../include/lldb/Core/PropertiesBase.td"
+
+let Definition = "platformwasm" in {
+  def RuntimePath : Property<"runtime-path", "FileSpec">,
+                    Global,
+                    DefaultStringValue<"">,
+                    Desc<"Path to the WebAssembly runtime binary.">;
+  def RuntimeArgs : Property<"runtime-args", "Args">,
```

Comment:
How do we order settings, if at all?

Wondering if this should be below PortArg because it's for everything but the port. Also would be good to add a note like "(for port related arguments, use port-arg instead)".